Grids AI Assistant: Add API Descriptions Part 2 (#8767)
DevExpress/devextreme-documentation: api-reference/40 Common Types/15 grids/ResponseStatusTexts/ResponseStatusTexts.md

module: common/grids
export: ResponseStatusTexts
type: Object
generateTypeLink: 
---
---
##### shortDescription

> Response texts for each status.

Reviewer: ```suggestion
Response messages for each status.
```

---

> This object can contain two string values: `success` and `failure`.

Reviewer 2: ```suggestion
This object can contain either `success` or `failure`.
```

I don't think it can contain both at the same time

Author: This is an object type, and it can contain both at the same time. This type is used in customizeResponseText callbacks (both described in this PR), where the command's status is not returned. As such, customers need to define both if they wish to specify custom messages for both success and failure statuses.

Reviewer 2: ```suggestion
This object can contain both `success` and `failure`.
```
